seleniumByTechnocredits/src/seleniumPractise/manisha/Assignment1.java

```diff
@@ -0,0 +1,39 @@
+package seleniumPractise.manisha;
+
+import org.openqa.selenium.Alert;
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+
+import util.Init;
+
+public class Assignment1 {
+	
+	public static void main(String[] args) throws InterruptedException {
+		
```

Comment:
Remove spaces

Reply:
Code is not properly formatted.Also mention proper comments for any specific operation so that reviewer can easily understand the logic